Lib/test/mp_preload_flush.py

```
@@ -0,0 +1,12 @@
+import multiprocessing
+import sys
+
+if __name__ == '__main__':
+    assert 'a' not in sys.modules
+    multiprocessing.set_forkserver_preload(['a'])
+    for _ in range(2):
+        p = multiprocessing.Process()
+        p.start()
+        p.join()
+else:
+    assert 'a' in sys.modules
```

There is a problem with this test, see
https://github.com/python/cpython/actions/runs/15558229444/job/43803979297?pr=135338

Each test should be written in such a way that `make -j test` command can be runned w/o errors.

---

Yes, for some reason it is failing to import the module on MacOS. Note the Ubuntu failure _looks like_ an intermittent race condition in resource tracker, unrelated to the MacOS failure, or indeed to this change at all.

Unfortunately I don't currently have access to a MacOS system, and it is difficult to debug without that. Fork server just swallows the exceptions on import failure, so there are no diagnostics available from the failed CI runs. I'm arranging to get access sometime over the next few days, which will allow me to dump those errors and debug it.

I considered pushing some additional diagnostics out to gather more information, but I didn't want to clutter the PR, confuse the history, and add extra load to CI.

Another option would be to wait for #126631 to be fixed, which would allow rewriting the test to just use `__main__` and drop creating and importing the module.

---

Turned out to be an easy fix once I had a working MacOS dev environment: it seems MacOS doesn't use `forkserver` by default...whoops. The new failure on one MacOS platform only looks unrelated: it is a failure in IDLE tests that AFAICT don't use multiprocessing at all.